extensions/wikia/Search/classes/Services/ESFandomSearchService.php

```diff
@@ -70,13 +72,16 @@ protected function consumeResponse( $response ) {
 		$results = [];
 
 		foreach ( $response as $item ) {
-			if ( isset( $item['_source'] ) ) {
+			if ( isset( $item['_source'] ) && isset( $item['_source']['title'] )
+			    && isset( $item['_source']['url'] ) ) {
 				$source = $item['_source'];
 
 				$results[] = [
 					'title' => html_entity_decode( $source['title'] ),
-					'excerpt' => isset( $source['excerpt'] ) ? html_entity_decode( $source['excerpt'] ) : '',
-					'vertical' => html_entity_decode( $source['vertical'] ),
+					'excerpt' => isset( $source['excerpt'] )
+						? html_entity_decode( $source['excerpt'] ) : '',
+					'vertical' => isset( $source['excerpt'] )
```

Comment:
copy/paste error, fixing.  going to create a bunch of constants to quiet SONAR as well